calcom/cal.com
# App/exchange v2

Reopens #2376 

## What does this PR do?

Adds Microsoft Exchange 2013 and 2016 to the app store.

This branch is created from `main` rather than the old `monorepo/app-store`

Fixes #2141

## Type of change




## How should this be tested?



- Go to the app store
- Go to the Microsoft Exchange 2013 & 2016 pages
- Click install app and the page to input your Exchange credentials should popup
